api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1066,6 +1067,10 @@ message Cluster {
 
     // Optional configuration for the RoundRobin load balancing policy.
     RoundRobinLbConfig round_robin_lb_config = 56;
+
+    // Optional configuration for the Deterministic Aperture load balancing policy.
+    envoy.extensions.load_balancing_policies.deterministic_aperture.v3.DeterministicApertureLbConfig
+        deterministic_aperture_lb_config = 57;
```

Comment:
I don't think this is the correct way to use load balancing extension. 

See https://github.com/envoyproxy/envoy/pull/23472 as a example.

We won't added new field to `Cluster` for specific load balancing policy. All load balancing extension should be configured by the `load_balancing_policy` (wrapper of `TypedExtensionConfig`).

Reply:
@wbpcode  I understand what you are trying to say. Unfortunately currently the only way to get the load balancer configuration in the `Factory` is via the `ClusterInfo` object. So we will have to convert the `loadBalancingPolicy` to `DeterministicApertureLbConfig`. Is there an example for that?

Reply:
@wbpcode I have made the suggested changes (from #23472). `doc` build still fails :(